apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx

```diff
@@ -164,6 +164,59 @@ const CopyMoveFiles = (): ReactElement =>
     },
   });
 
+enum FileDownloadStatus {
+  Downloaded = 'Downloaded',
+  Downloading = 'Downloading',
+  Failed = 'Failed',
+}
+interface IFileItem {
+  objectId?: string;
+  path?: string;
+  sizeInBytes?: number;
+  status?: FileDownloadStatus;
+  timestamp: Date;
+  title: string;
+  extension: string;
+}
+
+const GetFileDownloads = (): ReactElement =>
+  ApiWithTextInput<IFileItem[]>({
+    name: 'getFileDownloads',
+    title: 'Get File Downloads',
+    onClick: {
+      validateInput: () => {
+        return;
+      },
+      submit: {
+        withCallback: (input, setResult) => {
+          const callback = (error, files): void => {
```

Comment:
Right now our linting is kinda wacky and won't tell you to do this, but I Think we'd want the parameter types to be specified.